packages/syft/src/syft/core/node/new/credentials.py

```diff
@@ -43,7 +43,7 @@ def __eq__(self, other: Any) -> bool:
         return self.verify_key == other.verify_key
 
     def __repr__(self) -> str:
-        return str(self)
+        return "..." + str(self)[-5::]
```

Comment:
This is the public key so there is no need to hide it.

Reply:
This is mainly for making the repr, short for a nice UI, maybe I will add it later in the UserNodeView class, I will revert it here now